### PR TITLE
[FIX] #19237 - Translations for select setting in App Setting

### LINKED ISSUE #19237: Translations not working for Apps Select Fields
### Description:

The `i18nLabel` labels in App level setting are not getting translated for `SELECT` Fields 

### Steps to reproduce:

1. Add a sample app n add settings with translations

### Expected behavior:

I expect to see the translations defined in `i18n` folder in the app

### Actual behavior:

It only shows text in the `i18nLabel` instead of the actual translations

### Server Setup Information:

- Version of Rocket.Chat Server: 
- Operating System: 
- Deployment Method: 
- Number of Running Instances: 
- DB Replicaset Oplog: 
- NodeJS Version: 
- MongoDB Version:

### Client Setup Information

- Desktop App or Browser Version:
- Operating System:

### PR DESCRIPTION
**Pull request by murtaza98**
## Issue(s)

Fixes #19237
## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Hotfix (a major bugfix that has to be merged asap)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by d-gubert** [comment]
Hey @murtaza98 ! This change doesn't require modifications in the Apps-Engine?

**Comment by murtaza98** [comment]
Hey @d-gubert Nopes. Translation are handled on core  so no changes to apps engine will be required 😁

**Comment by d-gubert** [comment]
Ok! Can you provide an example of what the settings config in the app would look like for this to work?

**Comment by murtaza98** [comment]
Hi @d-gubert Here is a sample app zip file. Hope it helps u to understand the bug. 
[multi-select-translations-sample_0.0.1.zip](https://github.com/RocketChat/Rocket.Chat/files/5399521/multi-select-translations-sample_0.0.1.zip)

Btw, Sorry for the delay :see_no_evil:

File: client/admin/apps/AppSettings.js [hunk 1]
```diff
@@ -52,14 +52,30 @@ function AppSetting({ appSetting, onChange, value, ...props }) {
 	const label = (i18nLabel && tApp(i18nLabel)) || (id || tApp(id));
 	const hint = useMemo(() => i18nDescription && <MarkdownText content={tApp(i18nDescription)} />, [i18nDescription, tApp]);
 
+	let translatedValues;
+	if (values) {
```
Comment (lolimay) [inline client/admin/apps/AppSettings.js:56]:
```suggestion
	if (values?.length) {
```

**Review by lolimay — CHANGES_REQUESTED** [review]
LGTM overall 👍

**Review by d-gubert — APPROVED** [review]

**Comment by murtaza98** [comment]
Closing in favour of https://github.com/RocketChat/Rocket.Chat/pull/23908

**Pull request closed without merging**